crates/libs/windows/src/core/factory_cache.rs

```diff
@@ -44,6 +44,9 @@ impl<C: RuntimeName, I: Interface> FactoryCache<C, I> {
     }
 }
 
+unsafe impl<C, I> core::marker::Send for FactoryCache<C, I> {}
+unsafe impl<C, I> ::core::marker::Sync for FactoryCache<C, I> {}
```

Comment:
Can we document why `Send` and `Sync` are safe to implement for `FactoryCache`?

Reply:
Done.